gem/templates/springster/patterns/basics/social-media/social-media_with-label.html

```diff
@@ -1,22 +1,20 @@
 {% load wagtailcore_tags wagtailimages_tags wagtailsettings_tags static i18n %}
 {% get_settings %}
 <ul class="social-media-list">
-  <li class="social-media-list__item"><label class="social-media__label">{% trans "Share" %}:</label></li>
+  <li class="social-media-list__item"><label class="social-media__label">{% trans "Share" %}</label></li>
   <!--<li class="social-media-list__item">
```

Comment:
@projectmushroom we never investigated further, I'm not sure what was the consensus